doc/docs/modules/ROOT/pages/quickstart.adoc

```diff
@@ -195,6 +200,15 @@ Spark works with data in a tabular fixed schema. To accomplish this Neo4j Connec
 
 TK list of supported data types
 
+=== Consideration on the filters
+
+The Neo4j Spark Connector implements the SupportPushDownFilters interface, that allows you to push the Spark filters down to the Neo4j layer. In this way the data that Spark will receive will be already filtered by Neo4j.
```

Comment:
should probably mention that all filters are ANDed together?

Reply:
done at line 213